google/cloud/forseti/notifier/notifiers/cscc_notifier.py

```diff
@@ -200,15 +239,34 @@ def _send_findings_to_cscc(self, violations, organization_id=None,
 
         # beta api
         if source_id:
+            formatted_cscc_findings = []
             LOGGER.debug('Sending findings to CSCC with beta API. source_id: '
                          '%s', source_id)
-            findings = self._transform_for_api(violations, source_id=source_id)
+            new_findings = self._transform_for_api(violations,
+                                                   source_id=source_id)
 
             client = securitycenter.SecurityCenterClient(version='v1beta1')
 
-            for finding_tuple in findings:
-                finding_id = finding_tuple[0]
-                finding = finding_tuple[1]
+            findings_in_cscc = client.list_findings(source_id=source_id)
```

Comment:
It seems you are not paging through the results with the nextPage token.  Can you please do that?

Reply:
Verified that findings from all the pages are retrieved.